e2e-tests/insurance/cypress/e2e/journeys/eligibility/change-your-answers/change-your-answers-buyer-country.spec.js

```diff
@@ -0,0 +1,81 @@
+import { INSURANCE_ROUTES } from '../../../../../../constants/routes/insurance';
+import { INSURANCE_FIELD_IDS } from '../../../../../../constants/field-ids/insurance';
+import { completeAndSubmitBuyerCountryForm } from '../../../../../../commands/forms';
+import { summaryList, submitButton } from '../../../../../../pages/shared';
+import { country } from '../../../../../../fixtures/application';
+import mockCountries from '../../../../../../fixtures/countries';
+
+const { BUYER_COUNTRY } = INSURANCE_FIELD_IDS.ELIGIBILITY;
+
+const {
+  START,
+  ELIGIBILITY: { BUYER_COUNTRY_CHANGE, CHECK_YOUR_ANSWERS },
+} = INSURANCE_ROUTES;
+
+const oldCountry = country.NAME;
+const newCountry = mockCountries[0].NAME;
```

Comment:
Can we just check if not already that `mockCountries[0]` is not the same country as the country in the mock application? I suspect it might be 😄

Reply:
they're different.  mockCountrys[0] is Abu Dhabi